x-pack/plugins/cases/server/client/cases/utils.ts

```diff
@@ -443,22 +464,7 @@ export const getDurationForUpdate = ({
   status?: CaseStatuses;
 }): Pick<CaseAttributes, 'duration'> | undefined => {
   if (status && status === CaseStatuses.closed) {
-    try {
-      if (createdAt != null && closedAt != null) {
-        const createdAtMillis = new Date(createdAt).getTime();
-        const closedAtMillis = new Date(closedAt).getTime();
-
-        if (
-          !isNaN(createdAtMillis) &&
-          !isNaN(closedAtMillis) &&
-          closedAtMillis >= createdAtMillis
-        ) {
-          return { duration: Math.floor((closedAtMillis - createdAtMillis) / 1000) };
-        }
-      }
-    } catch (err) {
-      // Silence date errors
-    }
+    return getDuration({ createdAt, closedAt });
```

Comment:
nit: I think the logic could be reduced slightly by doing:

```
if (status === CaseStatuses.closed) { }
else if (status == CaseStatuses.open || status === CaseStatuses['in-progress']) {}
```

If `status === undefined` then the checks against the `CaseStatuses.*` wouldn't evaluate to `true`. Or you could keep the `status !== undefined` like:

```

if (status == null) { return }
else if (status === CaseStatuses.closed) { ... }
else { return { duration: null } }

```